### ISSUE TITLE
Manage columns labels gone

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1020]
![manage-columns](https://cloud.githubusercontent.com/assets/1663057/7908318/cecbe168-083b-11e5-88b0-662607a7a0f3.png)

**Comment by alexkalderimis** [comment]
CSS white on white maybe?

**Comment by julie-sullivan** [comment]
@alexkalderimis Doesn't look like it:

```
[13:11:21.964] "RENDER FAILED" [object Object] [object Error]
[13:11:21.965] TypeError: this.model.collection is undefined @ http://cdn.intermine.org/js/intermine/im-tables/2.0.0-beta/imtables.js:9317
```

**Comment by joshkh** [comment]
Fixed by pull request a55759e885d992c55d4b1e5c4180a94939979c18
